lookup_tables/greynoise/advanced/noise_advanced.yml

```diff
@@ -81,25 +74,16 @@ LogTypeMap:
       Selectors:
         - "ClientIP"
         - "OriginIP"
-    - LogType: Crowdstrike.AIDMaster
-      Selectors:
-        - "aip"
     - LogType: Crowdstrike.ActivityAudit
       Selectors:
         - "UserIp"
-    - LogType: Crowdstrike.CriticalFile
-      Selectors:
-        - "aip"
     - LogType: Crowdstrike.DetectionSummary
       Selectors:
         - "LocalIP"
         - "OriginSourceIpAddress"
     - LogType: Crowdstrike.DNSRequest
       Selectors:
         - "IpAddress"
```

Comment:
we might be missing a block here, at 5e9165bf6afccb8a0dbce1abefbc5f9c794db9c1 line 87-89 looked like 

```
  - LogType: Crowdstrike.AIDMaster
      Selectors:
        - "aip"
```